src/pages/index.js

```diff
@@ -56,48 +94,100 @@ const addCardPopUp = new PopupWithForm(
 );
 addCardPopUp.setEventListeners();
 
-/* -------------------------------------------------------------------------- */
-/*                                   Section                                  */
-/* -------------------------------------------------------------------------- */
+/* ---------------------------Popup Confirm Delete--------------------------- */
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (item) => {
-      console.log(item);
-      const cardEl = renderCard(item);
-      cardSection.addItem(cardEl);
-    },
-  },
-  ".cards__list"
+const confirmDeletePopup = new PopupWithConfirmation(
+  "#confirm-image-delete-popup",
+  handleDeleteConfirmSubmit
 );
-cardSection.renderItems();
+confirmDeletePopup.setEventListeners();
 
-const userInfoNew = new UserInfo(".profile__name", ".profile__description");
+/* ------------------------Change Profile Avatar Popup----------------------- */
+
+const changeProfileAvatarPopUp = new PopupWithForm(
+  "#change-profile-avatar-popup",
+  handleChangeProfileAvatarFormSubmit
+);
+changeProfileAvatarPopUp.setEventListeners();
+
+const userInfoNew = new UserInfo(
+  profileName,
+  profileDescription,
+  profileAvatar
+);
+
+api.fetchUserInfo().then((data) => {
+  let userData = data;
```

Comment:
Needs correction:use `const` keyword for the constant variables and there is no need to reassign the value here, just rename `data` to `userData`: `then(userData => `